PodcastGenerator/admin/episodes_edit.php

```diff
@@ -31,6 +31,7 @@
         unlink('../' . $config['img_dir'] . pathinfo('../' . $config['upload_dir'] . $_GET['name'], PATHINFO_FILENAME) . '.png');
     }
     generateRSS();
+    pingServices();
```

Comment:
Wouldn't it be better to embed this call into the `generateRSS()` function?

Reply:
It's better to have `generateRSS()` and `pingServices()` as separate functions, as they have different purposes. This also makes it possible to have a manual re-ping feature the same way as there's a manual regenerate RSS feature in the admin pages.